shared_model/bindings/model_query_builder.hpp

```diff
@@ -33,8 +33,10 @@ namespace shared_model {
       explicit ModelQueryBuilder(const proto::TemplateQueryBuilder<Sp> &o)
           : builder_(o) {}
 
+      void setDefaultValues();
```

Comment:
Don't really sure that it should be as a part of API
Imo it's better to inline it in the default constructor